addOns/commonlib/src/main/resources/org/zaproxy/addon/commonlib/internal/vulns/vulnerabilities.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8" ?>
-<vulnerabilities xmlns="http://tempuri.org/XMLSchemaOptions.xsd">
+<?xml version="1.0" encoding="utf-8"?>
+<vulnerabilities xmlns="https://tempuri.org/XMLSchemaOptions.xsd">
```

Comment:
HTTPS is not supported.

Reply:
Oh I didn't even try it, when I saw "tempuri" I figured it was a placeholder

Reply:
Yeah this doesn't seem to exist anymore. According to archive.org it did serve content at some point but now just redirects to Bing also https is offered but not for that DNS the cert offered is for *.oneroute.microsoft.com.

How about we change to https://zaproxy.org/vulnerabilities, it's just a unique name space it doesn't have to actually exist. (From my reading anyway.... 🤷‍♂️)

Reply:
Or just remove it? Fine either way though.

Reply:
Removed.